Plugin/CartIntercept.php

```diff
@@ -56,37 +55,37 @@ public function _gate(Cart $cart)
             return $cart;
         }
     }
+
     public function sendCart(Cart $cart, $storeId)
     {
         $customer = $cart->getCustomerSession()->getCustomer();
         $email = $customer->getEmail();
-        $this->client = $this->client->getClient(true, $storeId);
+        $client = $this->clientManager->getClient($storeId);
         if ($email || $anonymousEmail = $this->isAnonymousReady($storeId)) {
             $items = $this->_getItems($cart);
             $data = [
-                'email'             => $email,
-                'items'             => $items,
-                'incomplete'        => 1,
-                'message_id'        => $this->sailthruCookie->getBid(),
-            ];    
-            $email = $email ? $email : $anonymousEmail;
+                'email'      => $email,
+                'items'      => $items,
+                'incomplete' => 1,
+                'message_id' => $this->sailthruCookie->getBid(),
+            ];
             try {
-                $this->client->_eventType = "CartUpdate";
-                $items = $this->_getItems($cart);
```

Comment:
@lindsaycooper @silverman63 Same here

Reply:
@aingram probably remnants of old implementation